pkg/envoy/rds.go

```diff
@@ -53,26 +54,26 @@ func (l RouteDomainCheck) Run() outcomes.Outcome {
 		for _, virtualHost := range dynRouteCfg.GetVirtualHosts() {
 			for _, domain := range virtualHost.GetDomains() {
 				foundAnyRouteDomains = true
-				if l.Domain == "" {
+				if len(l.Domains) == 0 {
```

Comment:
could `l` be renamed to `check` for clarity? A lot of the other checks use this and `l` is a bit ambiguous

Reply:
Sure! Thanks!